contrib_bots/lib/yoda_bot.py

```diff
@@ -0,0 +1,129 @@
+# See readme-yoda-bot.md for instructions on running this code.
+from __future__ import print_function
+import unirest
```

Comment:
Add a comment for this dependency. Btw, any specific reason for using this over `requests`?